### PR TITLE
Update ci-cd-azure-dev-ops.md Added links to Microsoft docs on secrets

### PR DESCRIPTION
**Pull request by mattou07**
## Description

_What did you add/update/change?_

Added two links to the Microsoft documentation on how to set Secrets for your Azure DevOps pipeline.

## Type of suggestion

* [x] New content

## Deadline (if relevant)

_When should the content be published?_

Anytime!

### CODE REVIEW

**Comment by alina-tincas** [comment]
Hi @mattou07 thank you for the PR 😊 

We will take a look at it as soon as we can 🙌

**Review by jonat123 — COMMENTED** [review]
@mattou07 I think it is a good addition, does it make sense to also add it to all the other versions of deploy? 😊

File: 12/umbraco-deploy/getting-started/cicd-pipeline/ci-cd-azure-dev-ops.md [hunk 1]
```diff
@@ -30,6 +30,13 @@ Firstly a web deployment (#4), takes the packaged build artifact and deploys it,
 
 The final step (#5) is Umbraco Deploy specific - to call a function defined in the PowerShell script and trigger the extraction. Replace `ApiSecret` with `ApiKey` if you're using the deprecated API key setting instead.
 
+{% hint style="info" %} 
+The Microsoft docs contains useful information, if you are unsure on how to set secrets for your pipeline:
+[Set secret variables](https://learn.microsoft.com/en-us/azure/devops/pipelines/process/set-secret-variables?view=azure-devops&tabs=yaml%2Cbash)
+
+[Protecting secrets in Azure Pipelines](https://learn.microsoft.com/en-us/azure/devops/pipelines/security/secrets?view=azure-devops)
```
Comment (jonat123) [inline 12/umbraco-deploy/getting-started/cicd-pipeline/ci-cd-azure-dev-ops.md:37]:
```suggestion
The Microsoft docs contain useful information, if you are unsure of how to set secrets for your pipeline:
* [Set secret variables](https://learn.microsoft.com/en-us/azure/devops/pipelines/process/set-secret-variables?view=azure-devops&tabs=yaml%2Cbash)

* [Protecting secrets in Azure Pipelines](https://learn.microsoft.com/en-us/azure/devops/pipelines/security/secrets?view=azure-devops)
```

File: 10/umbraco-deploy/getting-started/cicd-pipeline/ci-cd-azure-dev-ops.md [hunk 2]
```diff
@@ -30,6 +30,14 @@ Firstly a web deployment (#4), takes the packaged build artifact and deploys it,
 
 The final step (#5) is Umbraco Deploy specific - to call a function defined in the Powershell script and trigger the extraction.
 
+{% hint style="info" %} 
+The Microsoft docs contains useful information, if you are unsure on how to set secrets for your pipeline:
+[Set secret variables](https://learn.microsoft.com/en-us/azure/devops/pipelines/process/set-secret-variables?view=azure-devops&tabs=yaml%2Cbash)
+
+[Protecting secrets in Azure Pipelines](https://learn.microsoft.com/en-us/azure/devops/pipelines/security/secrets?view=azure-devops)
```
Comment (jonat123) [inline 10/umbraco-deploy/getting-started/cicd-pipeline/ci-cd-azure-dev-ops.md:37]:
```suggestion
The Microsoft docs contain useful information, if you are unsure of how to set secrets for your pipeline:
* [Set secret variables](https://learn.microsoft.com/en-us/azure/devops/pipelines/process/set-secret-variables?view=azure-devops&tabs=yaml%2Cbash)

* [Protecting secrets in Azure Pipelines](https://learn.microsoft.com/en-us/azure/devops/pipelines/security/secrets?view=azure-devops)
```

**Comment by jonat123** [comment]
@mattou07 Just had a quick look again, and I had a thought that it would make sense to put the links in a bullet list :) 

let me know what you think :D

**Review by jonat123 — APPROVED** [review]
@mattou07 Thanks for making the changes :D 

Merging it in!

**Pull request merged**